PayKids/presentation/src/main/java/com/paykids/presentation/view/signIn/SignInProviderFragment.kt

```
@@ -0,0 +1,41 @@
+import com.paykids.presentation.base.BaseFragment
+import com.paykids.presentation.databinding.FragmentSignProviderBinding
+import com.paykids.presentation.utils.LoggerUtils
+import com.paykids.presentation.utils.UiState
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SignInProviderFragment : BaseFragment<FragmentSignProviderBinding>() {
+    private val signViewModel: SignInViewModel by viewModels()
+
+    override fun initView() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)